### DISCUSSION TITLE
Randomize ore geneartion

### DISCUSSION THREAD

**Discussion by lethargistic** [DerFrZocker/Ore-Control#24] [Q&A]
I have a public world see on my SMP so, I'm just looking to randomize ore generation on my server without actually changing the way ores generate.

Does changing any option directly related to ore generation in config already change that?
Or how do I achieve this?

**Comment by DerFrZocker** [comment]
You can minimal change the amounts of veins per chunk (+-0.01) to change the position of the ore with out changing the amount of ores being generated much.
